source/extensions/all_extensions.bzl

```diff
@@ -1,30 +1,22 @@
+load("@bazel_skylib//lib:dicts.bzl", "dicts")
 load("@envoy_build_config//:extensions_build_config.bzl", "EXTENSIONS", "WINDOWS_EXTENSIONS")
 
+# These extensions are registered using the extension system but are required for the core Envoy build.
+# The map may be overridden by extensions specified in envoy_build_config.
+_required_extensions = {
+    "envoy.transport_sockets.raw_buffer": "//source/extensions/transport_sockets/raw_buffer:config",
```

Comment:
WDYT about registering TLS also? I suppose it's not explicitly required but it seems like it pretty much should be? I could go either way.

Reply:
SGTM.